src/main/java/org/hobbit/core/components/AbstractCommandReceivingComponent.java

```diff
@@ -367,9 +399,11 @@ protected String createContainer(String imageName, String containerType, String[
      * @param envVariables
      *            environment variables that should be added to the created
      *            container
+     * @param netAliases
+     *            network aliases that should be added to the created container
      * @return the Future object with the name of the container instance or null if an error occurred
      */
-    protected Future<String> createContainerAsync(String imageName, String containerType, String[] envVariables) {
+    protected Future<String> createContainerAsync(String imageName, String containerType, String[] envVariables, String[] netAliases) {
```

Comment:
Why not simply overloading the method with the "old" method interface which is calling the new method with `null` for the aliases. The issue that I see now is that other people might not be aware of the possibility to use `null` instead of defining an alias :thinking:

Reply:
"Old" method interface currently only exists in develop branch and wasn't in any releases.